api/src/main/java/org/openmrs/api/impl/ConditionServiceImpl.java

```diff
@@ -114,6 +114,16 @@ public Condition getCondition(Integer conditionId) {
 		return conditionDAO.getActiveConditions(patient);
 	}
 
+	/**
+     * @see ConditionService#getAllConditions(Patient)
+	 * @since 2.4.0
```

Comment:
The since annotation should be only on the service interface, ConditionService

Reply:
@dkayiwa  I updated the PR.